### PR TITLE
Added levels to populate.

### PR DESCRIPTION
**Pull request by ppepos**
Added levels to populate...

Effect hasn't really changed but can be multiplied by a factor to increase the amount of dummy data

```
make populate level=2
```

Signed-off-by: Philippe Pepos Petitclerc ppeposp@gmail.com

### CODE REVIEW

**Comment by privat** [comment]
Why not just extending the existing db_loader with a level option?

What is the effect of a huge db? (cf #27 )

**Comment by ppepos** [comment]
reloled

File: src/db_loader.nit [hunk 1]
```diff
@@ -100,6 +106,14 @@ for player in players do
 		if status.stars.not_empty then status.status = "success"
 		config.missions_status.save status
 	end
+
+	# Spread some love (or friendships =( )
```
Comment (privat) [inline src/db_loader.nit:110]: https://www.youtube.com/watch?v=rslShTbqNbo

**Comment by privat** [comment]
+1

File: Makefile [hunk 2]
```diff
@@ -22,10 +22,11 @@ debug:
 	nitc src/app_debug.nit -m src/app_serial.nit -o bin/app
 
 populate:
+	# There are levels to this... try: `make mass_populate level=2`
```
Comment (lbajolet) [inline Makefile]: That comment needs an update

Comment (ppepos) [reply Makefile]: done.

**Comment by lbajolet** [comment]
+1

**Pull request merged**